.github/workflows/build.yml

```diff
@@ -1,5 +1,11 @@
 name: build
-on: ["push", "pull_request"]
+on:
+  pull_request:
+    branches:
+    - main
+  push:
```

Comment:
I suggest that the CI check of push be kept as it is, because it is possible that the docs cannot be built before merging.

Reply:
Now, CI will run the same job (build) twice, since every PR is merged into main branch.
So I copy konfig PPL, run this PPL when PR is created, and run again after PR is merged and push to main branch